packages/core/src/App/Containers/Layout/header/dtrader-header.jsx

```diff
@@ -0,0 +1,340 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withRouter, useHistory } from 'react-router-dom';
+import { DesktopWrapper, MobileWrapper, Text, Icon } from '@deriv/components';
+import { routes, isMobile, getDecimalPlaces, getPlatformInformation, platforms, PlatformContext } from '@deriv/shared';
+import { AccountActions, MenuLinks, PlatformSwitcher } from 'App/Components/Layout/Header';
+import platform_config from 'App/Constants/platform-config';
+import RealAccountSignup from 'App/Containers/RealAccountSignup';
+import SetAccountCurrencyModal from 'App/Containers/SetAccountCurrencyModal';
+import NewVersionNotification from 'App/Containers/new-version-notification.jsx';
+import { connect } from 'Stores/connect';
+import ToggleMenuDrawer from 'App/Components/Layout/Header/toggle-menu-drawer.jsx';
+import { AccountsInfoLoader } from 'App/Components/Layout/Header/Components/Preloader';
+import TempAppSettings from 'App/Containers/Layout/temp-app-settings.jsx';
+import { BinaryLink } from 'App/Components/Routes';
+import { Localize } from '@deriv/translations';
+
+const TradingHubMenuHomepage = () => {
+    const redirect = useHistory();
+    return (
+        <div
+            className='trading-hub-header__tradinghub'
+            onClick={() => {
+                redirect.push(routes.trading_hub);
+            }}
+        >
+            <Icon icon='IcAppstoreMenuHomepage' size={30} />
+        </div>
+    );
+};
+
+const Divider = () => <div className='header__menu--dtrader--separator' />;
+
+const RedirectToOldInterface = () => {
+    const platform_store = React.useContext(PlatformContext);
+    const disablePreAppstore = () => {
+        platform_store.setIsPreAppStore(false);
+    };
+    return (
+        <div className='trading-hub-header__dtrader--redirect'>
+            <BinaryLink
+                to={routes.trade}
+                className='trading-hub-header__dtrader--redirect--link'
+                onClick={disablePreAppstore}
+            >
+                <Text as='p' size='xs' color='general'>
```

Comment:
General is a default color of Text component
```suggestion
                <Text as='p' size='xs'>
```